core/arch/arm/plat-stm32mp1/conf.mk

```diff
@@ -1,5 +1,3 @@
-PLATFORM_FLAVOR ?= stm32mp157c-ev1
```

Comment:
Minor: since you removed the flavor completely, there is nowhere we can find which board exactly the default build will work on (er... is it even proper English syntax? but I think you get the point).

Reply:
ok, i get your point. i will restore.